README.md

```diff
@@ -5,3 +5,13 @@
 - async_stream_writer - takes the GRPC data chunks and writes them onto the GRPC stream.
 
 The reason for the division is to be able to utilize multiple CPU cores to process the different stages simultaneously.
+
+### Integration testing
+Goto src/test/integration and run:
+```
+ACCEPT_EULA=yes docker-compose up --build -d
```

Comment:
Usually we write `ACCEPT_EULA=<yes/no>` so people make an active decision to accept the eula and not just copy paste it